packages/@aws-cdk/aws-sqs/lib/queue.ts

```diff
@@ -277,7 +277,7 @@ export class Queue extends QueueBase {
       }
 
       if (encryption === QueueEncryption.KMS_MANAGED) {
-        const masterKey = kms.Key.fromKeyArn(this, 'Key', 'alias/aws/sqs');
+        const masterKey = kms.Key.fromKeyArn(this, 'Key', `arn:aws:kms:${Aws.REGION}:${Aws.ACCOUNT_ID}:alias/aws/sqs`);
```

Comment:
This grant isn't even necessary. The AWS-managed key will allow appropriate usage anyway.

Reply:
Makes sense and I've should have noticed this we are dealing with the default key here. :D

Thank you for fixing this!